drizzlepac/hapsequencer.py

```diff
@@ -622,6 +622,11 @@ def run_hap_processing(input_filename, diagnostic_mode=False, input_custom_pars_
                 log.warning("This Total Data Product only has Grism/Prism data and no direct images: {}".format(total_item.drizzle_filename))
                 log.warning("No SVM processing is done for the Grism/Prism data - no SVM output products are generated.")
                 product_list += [total_item.trl_filename]
+            
+            # Add skycell header keyword to SVM flt(c) files
+            # iterates for each total object, but passes full total object product list to the function
+            for image in total_item.edp_list:
```

Comment:
The comment needs to be updated.  The iteration is over each exposure in this total_item, but only the actual exposure name is provided to the function.

Reply:
comment updated.